Doc: Feedback #3070 - Add the `value` prop (#76)
vtex-apps/store-form: docs/README.md

| Prop name   | Type                                 | Description                                                                                                                                                                                                                                      | Default Value |
| ----------- | ----------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |

> | `pointer`   | `string`    | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) JSON schema pointer i.e. the JSON schema path  (for example: #/properties/firstName) in which the form block inputs should be validated against. | `undefined`              |

Reviewer: ```suggestion
| `pointer`   | `string`    | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) JSON schema path (e.g., `#/properties/firstName`) for validating form block inputs. | `undefined`              |
```

> | `inputType` | `enum` | Defines which type of a text field should be rendered: <br>`input`: renders a normal text field.<br>`hidden`: does not render any text field. It should be used in scenarios where you want to pre-define a field value to be submitted to the form, but that should be hidden (and, therefore, editable) to users. <br>`password`: renders a password text field.                 | `input`           |

Reviewer: ```suggestion
| `inputType` | `enum` | Defines the type of the text field. Possible values are: `input` - renders a regular text field; `hidden` - renders a hidden text field used for pre-defining an editable value to be submitted to the form; `password` -  renders a password text field.                 | `input`           |
```

> | `value` | `string` | Sets the `value` field when the `inputType` is `hidden`. | `undefined` |

Author: ```suggestion
| `value` (optional) | `string` | Sets the value of the `inputType` when it is `hidden`.  | `undefined` |
```

> | `value` (optional) | `string` | Sets a pre-defined text to be submitted when the `inputType` when is `hidden`.  | `undefined` |

Author: ```suggestion
| `value` (optional) | `string` | Displays a pre-defined text to be submitted when the `inputType` is set as `hidden`.  | `undefined` |
```

> | `label` | `string` |  ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Field's name when rendered | Property's title  |

Reviewer: ```suggestion
| `label` | `string` |  ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Label of the input field. | Property's title  |
```
